prefs/build.gradle

```diff
@@ -34,5 +34,5 @@ dependencies {
     implementation "com.google.dagger:hilt-android:$hilt_version"
     kapt "com.google.dagger:hilt-compiler:$hilt_version"
 
-    api 'com.github.tfcporciuncula:flow-preferences:1.3.1'
+    api 'com.github.tfcporciuncula:flow-preferences:1.4.0'
```

Comment:
I haven't checked the differences but there's already a v1.6.0

Reply:
Looks like they've changed the package ID

https://github.com/tfcporciuncula/flow-preferences